packages/models-library/src/models_library/api_schemas_directorv2/comp_tasks.py

```diff
@@ -18,6 +18,7 @@ class ComputationGet(ComputationTask):
     stop_url: AnyHttpUrl | None = Field(
         None, description="the link where to stop the task"
     )
+    model_config = ConfigDict(json_schema_extra={"examples": []})
```

Comment:
this looks wrong

Reply:
Q: why empty example? Either remove it or add an example, right?

Reply:
did you forget that one?

Reply:
Forgot to extend the examples here